File: Pattern18.java
// *********
//  *     *
//   *   *
//    * *
//     *

import java.util.Scanner;
public class Pattern18 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number:");
        int n = in.nextInt();
        for (int i = 1; i <= n; i++) {
            for (int k = 1; k <= i; k++) {
                System.out.print(" ");
            }
            int col = i == 1 ? 2 * n - 1 : n;
            for (int j = i; j <= col; j++) {
                if (col < 2 * n - 1) {
                    if (j == i || j == n) {
                        System.out.print("* ");
                    } else {
                        System.out.print("  ");
                    }
                } else {
                    System.out.print("*");
                }
            }
            System.out.print("\n");
        }
    }
}

File: PATTERNS/Pattern18/Pattern18.py
    # *********
    #  *     *
    #   *   *
    #    * *
    #     *

n = int(input("Enter the row number:"))

for i in range(1,n+1):
    for k in range(1,i+1):
        print(" ",end='')
    col = 2*n-1 if i==1 else n
    for j in range(i,col+1):
        if(col<2*n-1):
            if(j==i or j==n):
                print("* ",end='')
            else:
                print("  ",end='')
        else:
            print("*",end='')
    print()
